doc/GUIDE.md

```diff
@@ -987,13 +987,8 @@ A complete list of all the supported command line options is given below:
 | `--dump-includes <String>`                                   | dump included symbols into specified file (see below)        |
 | `--include-[function,constant,struct,union,typedef,var]<String>` | Include a symbol of the given name and kind in the generated bindings. When one of these options is specified, any symbol that is not matched by any specified filters is omitted from the generated bindings. |
 | `--version`                                                  | print version information and exit |
-
-**macOS platform options (available only when running on macOS):**
-
-| Option                       | Meaning                                                                                                                                                                                                                                 |
-|:-----------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `--mac-framework-dir <dir>`  | specify the framework directory dir include files <br/>defaults to the current Mac OS X SDK dir<br/> This removes the need of having a compile_flags.txt with the required `-framework XYZ` options in the folder where jextract is ran |
-| `-f, framework <framework>` | specify the name of the library, path will be expanded to that of the framework folder                                                                                                                                                  |
+| `--macos-framework <dir>` (macOs only)           | specify the framework directory include files <br/>defaults to the current Mac OS X SDK dir<br/> This removes the need of having a compile_flags.txt with the required `-framework XYZ` options in the folder where jextract is ran |
+| `-f <framework>` (macOs only)                    | specify the name of the library, path will be expanded to that of the framework folder                                                                                                                                                 |
```

Comment:
I don't think we should mention the workaround using `compile_flags.txt` here. This won't make sense to users who are not familiar with it, and after this change, there's no need for the workaround.